docs/packaging/updating-an-existing-package.md

```diff
@@ -167,6 +167,21 @@ Full release notes:
 - [ ] Package was built and tested against unstable
 ```
 
+### Other commit message format examples
```

Comment:
It might be better to have multiple subheadings for the above section. Something like:

```md
# Commit message formatting

There should be a meaningful summary line (which starts with the package name), a blank line, and then the rest of the commit message.

## Package update

{text from existing "commit message format" section}

## Other package change
```